molecule/driver/delegated.py

```diff
@@ -94,7 +102,7 @@ class Delegated(base.Base):
         platforms:
           - name: instance-vagrant
 
-    Provide the files Molecule will preserve post ``destroy`` action.
+    Provide the files Molecule will preserve post `destroy` action.
```

Comment:
Nothing major but I think we want the ` `` ` here since that is how Sphinx does code blocks (if that is what is intended).

Reply:
Fixed